srcs/go/utils/iostream/iostream.go

```diff
@@ -19,6 +20,10 @@ func Tee(r io.Reader, ws ...io.Writer) error {
 		}
 		for _, w := range ws {
 			fmt.Fprintln(w, string(line))
+			datas := strings.Split(string(line), ":")
```

Comment:
This packages is for generic utils functions, they shouldn't contain application logics.